src/main/java/com/jalasoft/jfc/controller/VideoConverterController.java

```diff
@@ -39,16 +40,29 @@
 @RequestMapping(path = "/videoConverter")
 public class VideoConverterController {
 
-    // Constant upload file.
-    private static final String UPLOADED_FOLDER = "src/main/java/com/jalasoft/jfc/resource/";
+    // Variable PathJfc type.
+    private PathJfc pathJfc;
 
-    // Constant path converted file.
-    private static final String CONVERTED_FILE = "src/main/java/com/jalasoft/jfc/resource/";
+    // Variable upload file.
+    private final String uploadedFile;
+
+    // Variable path converted file.
+    private final String convertedFile;
+
+    VideoConverterController(){
+        try {
+            pathJfc = new PathJfc();
+        }
+        catch (IOException ex){
+            ex.printStackTrace();
+        }
```

Comment:
check identation and verify if the exception type is the correct

Reply:
the exception is Correct